payjoin/src/core/receive/v2/mod.rs

```diff
@@ -213,17 +251,15 @@ impl State for UninitializedReceiver {}
 impl Receiver<UninitializedReceiver> {
     /// Creates a new [`Receiver<Initialized>`] with the provided parameters.
     ///
-    /// # Parameters
-    /// - `address`: The Bitcoin address for the payjoin session.
-    /// - `directory`: The URL of the store-and-forward payjoin directory.
-    /// - `ohttp_keys`: The OHTTP keys used for encrypting and decrypting HTTP requests and responses.
-    /// - `expire_after`: The duration after which the session expires.
+    /// This is the beginning of the receiver protocol in Payjoin v2. It uses the passed address,
+    /// store-and-forward Payjoin directory URL, and the OHTTP keys to encrypt and decrypt HTTP
+    /// requests and responses to initialize a Payjoin v2 session.
     ///
-    /// # Returns
-    /// A new instance of [`Receiver<Initialized>`].
+    /// Expiration time can be optionally defined to set when the session expires (due to
+    /// inactivity of either party, etc.).
```

Comment:
```suggestion
    /// Expiration time can be optionally defined to set when the session expires (due to
    /// inactivity of either party, etc.) or otherwise set to a default of 24 hours.
```

Reply:
We could make 2 constructors to clarify this 

```rs
pub fn create_session(expire_after: Duration); // instead of an Option
pub fn create_session_24h_default();
```